incubator/hnc/config/crd/patches/singleton-validation_unserve-v1alpha1_hierarchyconfiguration.yaml

```diff
@@ -0,0 +1,165 @@
+# Please remember to re-copy CRD from bases/ if there's a CRD change. See
+# https://github.com/kubernetes-sigs/multi-tenancy/issues/1035 for detail.
+#
+# The following "spec.versions" patch is first copied from the crd/bases/
+# directory generated by controller-gen. Then we manually make changes to it so
+# that the entire "spec.versions" field can be patched. Here are the changes:
+# 1) Add singleton metadata validation to each version, e.g.
+#            metadata:
+#              properties:
+#                name:
+#                  type: string
+#                  enum:
+#                    - hierarchy
+#              type: object
+# 2) Set v1alpha1 as unserved, e.g. "served: false", in v1alpha1 schema.
```

Comment:
Mostly the reviewers just need to review this part and the manual update part below.